model/rulefmt/rulefmt.go

```diff
@@ -233,6 +233,14 @@ func (r *RuleNode) Validate() (nodes []WrappedError) {
 				node: &r.Record,
 			})
 		}
+		// While record is a valid UTF-8 it's common mistake to put PromQL expression in the record name.
+		// Disallow "{}" chars.
+		if strings.Contains(r.Record.Value, "{") || strings.Contains(r.Record.Value, "}") {
+			nodes = append(nodes, WrappedError{
+				err:  fmt.Errorf("potential issue in the recording rule name; should it be in expr?: %s", r.Record.Value),
```

Comment:
Let's make it explicit in the error what the "issue" is, otherwise users might not understand why it's being triggered

```suggestion
				err:  fmt.Errorf("braces present in the recording rule name; should it be in expr?: %s", r.Record.Value),
```